packages/grid_client/src/modules/models.ts

```diff
@@ -584,6 +590,7 @@ class FilterOptions {
   @Expose() @IsOptional() @IsBoolean() hasGPU?: boolean;
   @Expose() @IsOptional() @IsBoolean() rentable?: boolean;
   @Expose() @IsOptional() @IsInt() @Min(1) rentedBy?: number;
+  @Expose() @IsOptional() @IsString() status?: NodeStatusFilter;
```

Comment:
please use `isEnum` instead of string and also check the transformation should be done. You will find an example here in models file

Reply:
please check this [example](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/grid_client/src/modules/models.ts#L172)